src/loaders/GenesJsonAsMatrixZarrLoader.js

```diff
@@ -4,8 +4,8 @@ import { AbstractLoaderError } from './errors';
 import LoaderResult from './LoaderResult';
 
 export default class GenesJsonAsMatrixZarrLoader extends JsonLoader {
-  constructor(params) {
-    super(params);
+  constructor(dataSource, params) {
+    super(dataSource, params);
 
     this.schema = genesSchema;
```

Comment:
So this overrides the schema setting in the base `JsonLoader`? Could we just add more types to `typesToSchema`?

Reply:
What do you mean here?  I don't think there's a schema set in `JsonLoader`